.editorconfig

```diff
@@ -11,12 +11,8 @@ insert_final_newline = true
 trim_trailing_whitespace = true
 
 [*.py]
-# Use 80 for Docstrings and comments.
 indent_size = 4
 max_line_length = 100
 
-[docs/**.txt]
-max_line_length = 80
-
```

Comment:
Unrelated update: We don't have `.txt` files under `docs/` that we want formatting applied to, and we use 100 columns for comments and docstrings in Python code.